### ISSUE TITLE
[Bug]: [0103]Tpch 1t test both on tke and eks report 'query all  has failed to  be executed for timeout(600s)'.

### ISSUE THREAD

**Issue by Ariznawlll** [matrixorigin/matrixone#13970]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

main

### Commit ID

eb1f5996792ecaa7714a3eae317dc8fc551cfa16

### Other Environment Information

```Markdown
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

**TKE：**

job url：https://github.com/matrixorigin/mo-nightly-regression/actions/runs/7386943259/job/20095549265
<img width="1054" alt="image" src="https://github.com/matrixorigin/matrixone/assets/108530700/df53afc3-a4e0-4cd4-8df2-ec6c2e4ee80a">
<img width="1363" alt="image" src="https://github.com/matrixorigin/matrixone/assets/108530700/5680f98b-e9a3-4fdc-87d6-964b46fab814">

logs：
http://175.178.192.213:30088/explore?panes=%7B%22sjT%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnamespace%3D%5C%22mo-nightly-regression-20240102%5C%22,%20pod%3D%5C%22nightly-regression-dis-tp-cn-4j26x%5C%22%7D%20%7C%3D%20%60%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%22now-24h%22,%22to%22:%22now%22%7D%7D%7D&schemaVersion=1&orgId=1

**EKS：**
job url:https://github.com/matrixorigin/mo-auto-test/actions/runs/7392936604/job/20112118284
<img width="1049" alt="image" src="https://github.com/matrixorigin/matrixone/assets/108530700/66729758-aac1-476a-bec9-25747d00d8a3">

<img width="989" alt="image" src="https://github.com/matrixorigin/matrixone/assets/108530700/b1836163-1b2f-470a-b9d7-2a2580a21f87">

log:
http://a32e0b4d7525042c2b28e6bd8d950a40-514639438.us-west-2.elb.amazonaws.com/explore?panes=%7B%22_D_%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D%7D%5D,%22range%22:%7B%22from%22:%22now-6h%22,%22to%22:%22now%22%7D%7D%7D&schemaVersion=1&orgId=1

### Steps to Reproduce

```Markdown
daily regression on tke.
performance test on eks.

if you have any problem, please wechat me.
```

**Comment by Ariznawlll** [comment]
二分查了一下正确性，确认出问题的commit是：eb1f5996792ecaa7714a3eae317dc8fc551cfa16

**Comment by jiangxinmeng1** [comment]
fixed by #14018

**Comment by Ariznawlll** [comment]
verified!
